misk-crypto/src/main/kotlin/misk/crypto/CiphertextFormatter.kt

```diff
@@ -0,0 +1,74 @@
+package misk.crypto
+
+import com.amazonaws.encryptionsdk.internal.EncryptionContextSerializer
+import okio.ByteString
+import java.nio.ByteBuffer
+import okio.ByteString.Companion.toByteString
+import java.util.Arrays
+
+/**
+ * This class contains utility functions to serialize the deserialize the ciphertext
+ * generated by the [misk.crypto.CryptoModule] extension functions.
+ */
+internal sealed class CiphertextFormatter {
+  companion object {
+    /**
+     * Serialize the encryption context associated with the ciphertext and wrap both in a ByteString
+     * in the following format:
+     *
+     * `ciphertext length | ciphertext [ encryption context length | encryption context ]`
```

Comment:
When we are decrypting data, how do we differentiate ciphertext without this change (i.e. created before this PR was merged) from ciphertext with this change?

Is there a unit test to make sure that we can always properly handle the ciphertext format before this change is merged?